ibis/expr/datatypes/core.py

```diff
@@ -386,17 +386,21 @@ def _pretty_piece(self) -> str:
         args = []
 
         if (precision := self.precision) is not None:
-            args.append(f"prec={precision:d}")
+            args.append(str(precision))
 
         if (scale := self.scale) is not None:
-            args.append(f"scale={scale:d}")
+            args.append(str(scale))
 
         if not args:
             return ""
 
         return f"({', '.join(args)})"
 
 
+Numeric = Union[Integer, Floating, Decimal]
```

Comment:
How about adding a common baseclass instead?

Reply:
Done! Thanks for the review!